crates/quinn_test_client/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "quinn-test-client"
```

Comment:
I we're going to merge this, I'd prefer a more generic 'network test' name instead `quinn` specific.

Reply:
We should probably merge it into `wcn_operator` cli

Reply:
Sounds good